### ISSUE TITLE
Only 2 Entities

### ISSUE THREAD

**Issue by klehnst** [Skipperro/pajgps-homeassistant#3]
I have PAJ Vehicle Finder 4G 1.0 v1.
For me only 2 entities are displayed, once speed in kmh and under diagnosis (present/absent) from the PAJ GPS,
Unfortunately the coordinates cannot be seen.

**Comment by Skipperro** [comment]
Yes, this seems to be right for devices without battery:
![image](https://github.com/Skipperro/pajgps-homeassistant/assets/20705647/4303a590-2c26-47ca-8adc-7300ce28d2b8)
One is for speed and another will tell you in which area the GPS device is. Away/Abwesend means - outside any defined area.

The coordinates are a part of the device-tracker attributes:
![image](https://github.com/Skipperro/pajgps-homeassistant/assets/20705647/bd719c97-7a60-4643-a90e-a81c512a3786)

So you can use this device and select it on the list of devices to track on the map. In my case I created a "person" for my car and attached a tracker device to it:
![image](https://github.com/Skipperro/pajgps-homeassistant/assets/20705647/46fc6d77-700b-4a18-918b-92e45359f4b0)

Works great:
![image](https://github.com/Skipperro/pajgps-homeassistant/assets/20705647/19a75da1-6118-497a-9016-ebee5b89232d)

If you want I could add coordinates as numbers to the list of entities, but could you explain me what is it you want to achieve by having them as numbers?

**Comment by klehnst** [comment]
Thank you for the information and sorry for your wasted time.
It was my mistake, I didn't know that the coordinates were there.

**Comment by Skipperro** [comment]
It's not wasted time. Now I know that it would probably make sense to add a more detailed info to the ReadMe file on how to use map card. Or a documentation link at least. 
https://www.home-assistant.io/dashboards/map/

Yes, it's a different integration, but the map card is a standard component in Home Assistant so you don't have to install anything else.

Here is a quick tutorial on how to replicate the map I have in my dashboard:

1. While on Dashboard, enable Dashboard Edit (°°° menu in top-right corner)
![image](https://github.com/Skipperro/pajgps-homeassistant/assets/20705647/0382d0d5-6506-4f56-97bb-a621b6402b6e)
2. Add a new card and set it to "Panel (1 card") mode:
![image](https://github.com/Skipperro/pajgps-homeassistant/assets/20705647/0015ca05-126c-473a-a5a3-07f2e28935ea)
3. Add a card of the "Map Card" type and configure it as you like. For me default zoom of 12 and tracking of last 5 hours works best. Don't forget to add entities to track - in your case that would be your PAJ GPS Tracker.
![image](https://github.com/Skipperro/pajgps-homeassistant/assets/20705647/b4cba949-038a-47ef-986e-5db3dbacda89)

Let me know if that works for you. :smile:

**Comment by klehnst** [comment]
Nice! It works! Thank you for your time.
Is it possible to change the timespan like the energy dashboard? That would be really cool.

I think that guide should be insert to the description. that will be good for newbees like me.

**Comment by klehnst** [comment]
![image](https://github.com/Skipperro/pajgps-homeassistant/assets/46057062/e652e1a2-d7e1-4567-9a49-771ec52ac70b)

Yes i mean the time in which points on the map are displayed but like the Energydashboard, in the energy dashboard you can select the date and times, then you can see the energy production/consumption for the desired day/time

**Comment by Skipperro** [comment]
I don't think that standard map card supports this and history can give you history of speed and area you were located. I haven't saw the option to view points from specific period, but the raw data should be somewhere inside the HA, so.... maybe someone made some extended version of map card that is available as a plugin? 

If you'll find it, please let me know, I would also like to have such feature in my dashboard.
